src/Bundle/EventListener/ORMTranslatableListener.php

```diff
@@ -69,7 +69,7 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs): void
         }
     }
 
-    public function postLoad(LifecycleEventArgs $args): void
+    public function postLoad(PostLoadEventArgs $args): void
```

Comment:
```suggestion
    public function postLoad(LifecycleEventArgs|PostLoadEventArgs $args): void
```

or we can even skip the change in function notation. As we are reducing the scope of what is passed and this function is public, we should still allow for the old notation (until we support Doctrine 2.0)